source/core/module_coreHelpers.js

```diff
@@ -56,17 +56,21 @@
             fpSync = fn;
         };
 
-        Module.coreHelpers.utf8ArrayToStringWithNull = function (u8Array, idxIn) {
+        // Takes Vireo Strings that are UTF-8 encoded strings with known length and returns a JS string
+        // TODO mraj assumes valid UTF8 encoding https://github.com/ni/VireoSDK/issues/283
```

Comment:
Can we use https://github.com/mathiasbynens/utf8.js that seems to throw an error on malformed UTF-8 strings?